packages/wonder-blocks-core/components/unique-id-provider.js

```diff
@@ -66,10 +66,9 @@ export default class UniqueIDProvider extends React.Component<Props> {
         const {children, mockOnFirstRender, scope} = this.props;
 
         // If this is our first render, we're going to stop right here.
+        // Note: `firstRender` will be `false` on the first render if this
+        // component is a descendant of a `NoSSR`.
         if (firstRender) {
-            // We'll be needing this on the next render, so let's set it up.
-            this._idFactory = new UniqueIDFactory(scope);
-
             if (mockOnFirstRender) {
```

Comment:
I never liked this prop name; just went with it since it is how the ADR talked about it and is consistent in like components. Should really be `mockIfInitialSSRCompatibleRender` or something.